### DISCUSSION TITLE
[QUESTION]:daemonsets配置config_update_interval不生效

### DISCUSSION THREAD

**Discussion by onejimmyboy** [alibaba/ilogtail#1345] [Help] [answered]
version：1.8.4
配置config_update_interval不生效
![image](https://github.com/alibaba/ilogtail/assets/57432151/96e5d24a-d266-4f13-bda8-611e68667f11)
![image](https://github.com/alibaba/ilogtail/assets/57432151/358e62da-4084-4a7c-88f2-b924b9b45e21)

![image](https://github.com/alibaba/ilogtail/assets/57432151/c4481b30-738a-4222-9015-3ee06faf5eb3)

**Comment by messixukejia** [comment]
比较的时间应该是打印user yaml config file loaded跟配置文件变更的时间吧，而不是上次LogInput Resume:success的时间。

**Reply by onejimmyboy** [reply]
并不是这样啊，我试了一下不修改config_update_interval这个参数的话，查看user_yaml_config_config.d中的yaml文件的时候，发现并不是10s后更新配置。我理解的是如果修改了configmap的话，默认的情况下，最长10秒钟会看到user_yaml_config_config.d目录下的文件也更新，是大致看了一下将近1分钟的时候才更新

**Reply by messixukejia** [reply]
1 提供下版本号

2 配置yaml修改时间，日志中load打印的信息

**Reply by onejimmyboy** [reply]
version：1.8.4
![image](https://github.com/alibaba/ilogtail/assets/57432151/cb03c79e-ea64-44dd-aad5-abab115b6332)
![image](https://github.com/alibaba/ilogtail/assets/57432151/aa90c9af-3896-4f34-94b7-aa216d9f5da6)

**Reply by messixukejia** [reply, marked as answer]
![image](https://github.com/alibaba/ilogtail/assets/13250519/50aa13a9-3771-41af-bcf3-9943bd8a2331)
![image](https://github.com/alibaba/ilogtail/assets/13250519/2e85fe28-9c6a-4519-8645-ac76a55b5df2)
1、修改user_yaml_config.d/test.yaml，并查看执行时间
2、查看user_yaml_config.d/test.yaml被加载的日志。

**Reply by onejimmyboy** [reply]
你上面测试的是虚拟机部署的吧，而不是k8s部署的吧，k8s部署的时候user_yaml_config.d中的配置文件来源于configmap

**Reply by messixukejia** [reply]
logtail识别的是本地文件。configmap的同步时间建议查下k8s的文档吧
